applications/serial_lte_modem/src/slm_at_commands.c

```diff
@@ -217,9 +217,9 @@ static int handle_at_slmuart(enum at_cmd_type type)
 		case 460800:
 		case 921600:
 		case 1000000:
-			k_delayed_work_init(&slm_work.work, set_uart_wk);
+			k_work_init_delayable(&slm_work.work, set_uart_wk);
```

Comment:
If this can end up being called while the entry is pending or in progress, this could lead to unexpected behavior. 
In that case, it'd be better to initialize it in an init function, or use the `K_WORK_DELAYABLE_DEFINE` macro

Reply:
Indeed, this is obviously wrong, will fix.

Reply:
Fixed.